src/pages/Register.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import logo from './../assets/logo.png';
+
+function Register() {
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+    phoneNumber: '',
+  });
+
+  const [responseMessage, setResponseMessage] = useState('');
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    try {
+      const response = await fetch('https://cadr-pg.github.io/users/register', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        const data = await response.json();
+        setResponseMessage(`Success: ${data.message}`);
+      } else {
+        const errorData = await response.json();
+        setResponseMessage(`Error: ${errorData.message}`);
+      }
+    } catch (error) {
+      setResponseMessage(
+        `Error: ${error instanceof Error ? error.message : 'An unknown error occurred'}`,
+      );
+    }
+  };
```

Comment:
to do usuniecia. do wysylania requestow stworzymy oddzielnego clienta gdy backend juz bedzie gotowy.